products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,380 @@
+loaded in Memgraph, we are ready to gain some information out of it.
+
+
+1) Let's say you want to find 20 most expensive transfers.
+As mentioned before, transfers fees are represented in millions of euros.
+
+```opencypher
+MATCH 
+    (t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE 
+    t.fee is NOT NULL
+RETURN ROUND(t.fee) + 'M €' as transfer_fee, p.name AS player_name
+ORDER BY t.fee DESC LIMIT 20
+```
+
+2) What about finding the most expensive transfer per season? 
+
+```opencypher
+MATCH 
+    (s:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(:Player)
+WHERE 
+    t.fee is NOT NULL
+WITH s.name as season_name, MAX(t.fee) as max_fee
+RETURN ROUND(max_fee) + 'M €' as max_transfer_fee, season_name
+ORDER BY max_fee DESC
+```
+
+3) How about finding out which teams your favorite player has played for?
+If you wish to check the teams for another player, replace "Sime Vrsaljko"
+with the name of your favorite player.
+
+```opencypher
+MATCH 
+    (player:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[]-(team:Team)
+WHERE 
+    player.name = "Sime Vrsaljko"
+WITH DISTINCT team
+RETURN team.name AS team_name
+```
+
+You might wonder why we haven't specified a direction in our Cypher traversal with `(:Transfer)-[]-(:Team)`. 
+As we want to find the teams that player was transferred from (`(:Transfer)<-[]-(:Team)`) and transferred to
+(`(:Transfer)-[]->(:Team)`), we want to collect both inbound and outbound connections. In order to do so, we
+omit the arrow (`>`, `<`) in our Cypher command.
+
+4) Find players that were transferred to and played for FC Barcelona and
+count them by the player game position.
+
+```opencypher
+MATCH 
+    (team:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
+WHERE 
+    team.name = "FC Barcelona"
+WITH DISTINCT player
+RETURN player.position as player_position, COUNT(player) AS position_count, collect(player.name) as player_names
+ORDER BY position_count DESC
+```
+
+5) Football has seen a lot of rivalries develop between clubs during its rich and long history.
+One of the most famous ones is between fierce rivals FC Barcelona and Real Madrid.
+There is a term, El Clasico, for a match between those two teams. Let's find all the transfers between
+FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH 
+    (m:Team)-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team)
+WHERE
+    (m.name = "FC Barcelona" AND n.name = "Real Madrid") OR
+    (m.name = "Real Madrid" AND n.name = "FC Barcelona")
+MATCH
+    (t)<-[:TRANSFERRED_IN]-(p:Player)
+RETURN m.name as transferred_from_team, p.name as player_name, n.name as transfered_to_team
+```
+
+6) FC Barcelona is one of the most valuable football clubs in the world. Players often want to play there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH 
+    (m:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:TRANSFERRED_TO]->(n:Team)
```

Comment:
```suggestion
    (m:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player),
    (t)-[:TRANSFERRED_TO]->(n:Team)
```

As far as I see you are splitting them one query per line (which is good), but this was not consistent then.

Reply:
:+1: